exercise/templates/exercise/staff/results.html

```diff
@@ -10,45 +11,185 @@
 <li class="active">{% trans "All results" %}</li>
 {% endblock %}
 
+{{% block scripts %}
+{{ block.super }}
+<!--Library to handle multiselection on select tags-->
+<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-multiselect/0.9.15/css/bootstrap-multiselect.css">
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-multiselect/0.9.15/js/bootstrap-multiselect.min.js"></script>
+
+<!--Required libraries for exporting table data to xlsx, csv or txt.-->
+<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/TableExport/5.2.0/css/tableexport.min.css">
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/xlsx/0.14.3/xlsx.core.min.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/FileSaver.js/1.3.8/FileSaver.min.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/TableExport/5.2.0/js/tableexport.min.js"></script>
+
+<!--Custom for loading animation (loading_animation.css) and other extra customizations (results_staff.css)
+    results_staff.js is the main script for this page. -->
+<link rel="stylesheet" href="{% static 'exercise/css/loading_animation.css' %}" />
+<link rel="stylesheet" href="{% static 'exercise/css/results_staff.css' %}" />
+<script src="{% static 'exercise/results_staff.js' %}"
+        allExercisesUrl="/api/v2/exercises/"
+        exercisesUrl="/api/v2/courses/{{course.id}}/exercises"
```

Comment:
These require url reverse resolving. No hardcoding urls in the files :)

Reply:
How would I find the url reversers for them all? I can only find url for course api

`courseUrl="{% url 'api:course-detail' version=2 course_id=course.id %}"`

And still would have to do `courseUrl + "students"`

Reply:
I think you need to find the view class in the code and look for some URL key parameter that becomes the URL name for the URL resolver (maybe like this https://github.com/Aalto-LeTech/a-plus/blob/master/exercise/api/views.py#L66). The name is used in the static template tag.

In Python code, there is the function `api_reverse` that likewise uses the URL names to reverse them. 
https://github.com/Aalto-LeTech/a-plus/blob/master/aplus/api/__init__.py#L4

Reply:
Basenames are here: https://github.com/Aalto-LeTech/a-plus/blob/master/api/urls_v2.py . Url name is `api:basename-[list|detail]` depending if it's the list or detail version of the view handler.

```
exercisesUrl="{% url 'api:course-exercises-list' version=2 course_id=course.id %}"
studentsUrl="{% url 'api:course-students-list' version=2 course_id=course.id %}"
```

and so on

Reply:
Changed these.